ruby/lib/ci/queue/circuit_breaker.rb

```diff
@@ -46,7 +46,7 @@ def message
         private
 
         def current_timestamp
-          Time.now.to_i
+          CI::Queue::Time.now.to_i
```

Comment:
```suggestion
          CI::Queue.time_now.to_i
```

Would be a nicer interface IMO.